packages/@aws-cdk/aws-elasticloadbalancing/lib/load-balancer.ts

```diff
@@ -57,6 +57,11 @@ export interface LoadBalancerProps {
    * @default true
    */
   readonly crossZone?: boolean;
+
+  /**
+   * @default - Private subnets
```

Comment:
Please document with a description. Also, the `@default` mentioned is not correct. It should be `Public subnets if internetFacing, Private subnets otherwise`.